ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/structured_configs/inband-mgmt-parent-vrf.yml

```diff
@@ -78,7 +104,27 @@ management_api_http:
   enable_vrfs:
   - name: MGMT
   enable_https: true
+ntp:
+  local_interface:
+    name: null
+    vrf: null
```

Comment:
This looks wrong

Reply:
Looks like it's coming from [here](https://github.com/aristanetworks/avd/blob/71a101e7f722c855354d09e85578528ad5a368ad/python-avd/pyavd/_eos_designs/structured_config/base/ntp.py#L48-L54) due to